trashtalk/google_sheets.py

```diff
@@ -8,17 +8,16 @@
 from oauth2client.service_account import ServiceAccountCredentials
 
 from trashtalk.models import Cleanup, db_session
-from trashtalk.master_keys import MasterKeys
-key_chain = MasterKeys()
+from flask import Blueprint, render_template, request, flash, current_app
 
 scope = ['https://spreadsheets.google.com/feeds/'] #Sheet must be shared with Cloud account in order to find it
```

Comment:
Lets move this to `constants.py` and give it a more descriptive name. I don't know what `scope` is supposed to refer.

Reply:
1. **Scope** is a google docs variable. Imagine the location of a google sheet has a *file path*. **Sheet Key** is the *file name.* Google still needs **Scope** to act like a *file path,* to realize it needs to look in a certain data feed. The catch is that every user needs to create his own sheet that appears in his own google doc account. The **Trashtalk** sheet currently lives on my drive account. 

2. I think [settings](https://github.com/openoakland/TrashTalk/blob/master/trashtalk/settings.py), next to the GOOGLE_MAPS_ENDPOINT makes more sense than [constants](https://github.com/openoakland/TrashTalk/blob/master/trashtalk/settings.py). They both serve as url addresses for google services.

3. I'll improve the description